server/loginserver.ts

```diff
@@ -12,24 +12,22 @@
 const LOGIN_SERVER_TIMEOUT = 30000;
 const LOGIN_SERVER_BATCH_TIME = 1000;
 
+// @ts-ignore
```

Comment:
Please add a comment about what is being `@ts-ignore` / why its necessary. Are you sure you want a `@ts-ignore` here and not `// tslint:disable-next-line no-var-requires`?

Reply:
Or is this to avoid some having to `ts-ignore` on usage instead? Ignoring at such a high level here seems dangerous.

Reply:
Yeah, that's probably better.